Assets/_Application/DataTypeAdapters/Tiles3DImportAdapter.cs

```diff
@@ -0,0 +1,73 @@
+using UnityEngine;
+using UnityEngine.Events;
+using Netherlands3D.DataTypeAdapters;
+using Netherlands3D.Functionalities.OGC3DTiles;
+using System.IO;
+using Newtonsoft.Json;
+using Netherlands3D.Credentials;
+
+namespace Netherlands3D.Twin.DataTypeAdapters
+{
+    [CreateAssetMenu(menuName = "Netherlands3D/Adapters/Tiles3DImportAdapter", fileName = "Tiles3DImportAdapter", order = 0)]
+    public class Tiles3DImportAdapter : ScriptableObject, IDataTypeAdapter
+    {
+        [SerializeField] private Tile3DLayerGameObject layerPrefab;
+        [SerializeField] private UnityEvent<string> displayErrorMessageEvent;
+
+        public void Execute(LocalFile localFile)
+        {
+            var layerParent = GameObject.FindWithTag("3DTileParent").transform;
+            var newObject = Instantiate(layerPrefab, Vector3.zero, layerPrefab.transform.rotation, layerParent);
+
+            var layerComponent = newObject.gameObject.GetComponent<Tile3DLayerGameObject>();
```

Comment:
newObject is al een gameObject, dus de extra stap is niet nodig. Sterker nog: kan je met instantiate niet een implicit cast doen naar Tile3DLayerGameObject omdat dat de prefab type is?

Reply:
Dit is meer een nitpick dus ga ik resolven